tensorboard/pip_package/pip_smoke_test.sh

```diff
@@ -174,14 +200,15 @@ test_access_url() {
 }
 
 TEST_URL_FAILED=0
-test_access_url "http://localhost:${TEST_PORT}/data/logdir" || TEST_URL_FAILED=1
-test_access_url "http://localhost:${TEST_PORT}" || TEST_URL_FAILED=1
+test_access_url "${TB_URL}/data/logdir" || TEST_URL_FAILED=1
+test_access_url "${TB_URL}" || TEST_URL_FAILED=1
 
 echo
 echo "Terminating tensorboard binary at pid ${TB_PID}"
 echo
 
 kill -9 "${TB_PID}"
+wait "${TB_PID}" 2>/dev/null || true  # Wait to suppress "Killed" message.
```

Comment:
Cool. How does this line work? :)

Reply:
https://stackoverflow.com/questions/5719030/bash-silently-kill-background-function-process

Normally when you kill a background job that's attached to a Bash session you get a "[PID] Killed" message.  In this case it's a bit weird to see that even when the test succeeded, so adding the "wait" basically has the effect of rejoining the background job to the main Bash thread just before it's killed (since the signal sent by `kill` still needs to reach the job and shut it down) and suppressing that message.

I admit the margin is a little too narrow for the line comment to contain this explanation.  I could add a link to the SO question if you think it'd help explain?

Reply:
Got it. Thanks! I actually think that makes sense now.